deploy/cephcsi/image/Dockerfile

```diff
@@ -1,9 +1,7 @@
-FROM ceph/ceph:v14.2
+FROM ceph/ceph:v14.2.2-20190826
 LABEL maintainers="Ceph-CSI Authors"
 LABEL description="Ceph-CSI Plugin"
 
-# Removing ceph-iscsi repo to workaround the repo issue while upgrading
-RUN rm -f /etc/yum.repos.d/ceph-iscsi.repo && yum -y update && yum clean all
```

Comment:
@humblec any reason why ` yum update` is removed?

Reply:
@Madhu-1 I think we can keep it, updated the patch set.